### PR TITLE
Added remove buffers stage

### PR DESCRIPTION
**Pull request by leerichard42**

### CODE REVIEW

File: bin/gltf-pipeline.js [hunk 1]
```diff
@@ -7,6 +7,8 @@ var addDefaults = require('../').addDefaults;
 var removeUnusedImages = require('../').removeUnusedImages;
 var removeUnusedSamplers = require('../').removeUnusedSamplers;
 var removeUnusedShaders = require('../').removeUnusedShaders;
+var removeUnusedTechniques = require('../').removeUnusedTechniques;
```
Comment (pjcozzi) [inline bin/gltf-pipeline.js]: As discussed offline, we'll merge #15 first, but I will still review this PR now.

**Comment by pjcozzi** [comment]
Looks great!  No changes needed.

**Comment by leerichard42** [comment]
Should I do anything with this branch?

**Comment by pjcozzi** [comment]
No, I merged in master and fixed the conflicts.  This is good!

**Pull request merged**